testsuite/tests/afl-instrumentation/afl-showmap-test.run

```diff
@@ -11,8 +11,8 @@ echo "running $NTESTS tests..."
 for t in `seq 1 $NTESTS`; do
   printf "%14s: " `./test name $t`
   # when run twice, the instrumentation output should double
-  afl-showmap -m none -q -o output-1 -- ./test 1 $t
-  afl-showmap -m none -q -o output-2 -- ./test 2 $t
+  OCAMLRUNPARAM="d=8,N=256k" afl-showmap  -q -o output-1 -- ./test 1 $t
```

Comment:
I'd change this to `d=1`. Non-determinism introduced by multi-threaded execution will affect AFL stability. @jmid

Reply:
These are still just running the old single-threaded tests so one domain should be sufficient.